doc/source/user/how-to-index.rst

```diff
@@ -309,6 +309,24 @@ result as dimensions with size one::
     array([[[2, 2, 2, 2, 2]],
     <BLANKLINE>
            [[2, 2, 2, 2, 2]]])
+	   
+To get the indices of each maximum or minimum value for each (N-1)-dimensional array in an N-dimensional array, use :meth:`reshape` to reshape the array to a 2D array, apply argmax or argmin along ``axis=1`` and use :meth:`unravel_index` to recover the index of the values per slice::
```

Comment:
I considered this but opted not to because they were already linked in the same section. For future reference, is it always necessary to link to methods that have already been linked to within a few lines?

Reply:
It is always recommended because it increases accessibility on smaller screens where you have to scroll quite a bit even within a section.